### DISCUSSION TITLE
Domain Vercel Api

### DISCUSSION THREAD

**Discussion by yanivoxy** [vercel/next.js#51587] [Help]
### Summary

When building a multi-tenancy app with Vercel and Nextjs.  We are using the vercel API to add the domains to Vercel and middleware in Nextjs to display the domain.

When adding the domain to vercel to my project do I add an A record pointing to my website ip or vercels ip and nameservers? 
For instance, do i point to example.com > example.ourcompany.com or example.com > vercel.com (NameServers/A Record)

**Comment by IsiteYves** [comment]
@yanivoxy To use nameservers of your domain on vercel, you just copy the nameservers from vercel and paste them into the `nameservers` fields of your domain settings (on domain's `provider` company website like `GoDaddy`,`Namecheap`,etc).
1. Go to your project `Settings` tab on vercel
2. Go to `Domains`
3. Add new domain, copy nameservers and paste them where you bought the domain

**Comment by yanivoxy** [comment]
That wasn't the question. If a user wants to add his domain name does he
point it to vercel or my websites domain? Then use the vercel api to add
the domain to the project.

**Reply by nabildroid** [reply]
Vercel's and you claim it by calling the api
